src/backend/tests/models/test_charge.py

```diff
@@ -1,12 +1,27 @@
 import unittest
 
-from datetime import date
+from datetime import date, datetime, timedelta
 from dateutil.relativedelta import relativedelta
 from expungeservice.models.disposition import Disposition
 from tests.factories.charge_factory import ChargeFactory
 from tests.factories.case_factory import CaseFactory
 
 
+class ChargeTypeTestsParent(unittest.TestCase):
```

Comment:
Cool, I have not thought to do this yet. This seems smart. I have not thought this through yet, but do we also need to run a teardown per class?